docs/documentation.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Documentation
+---
+
+Once you’re ready to release your webhook feature, you’ll need to add documentation.
+At Svix we consider our documentation as part of the product.
+We put a lot of effort into making sure our documentation makes it as easy as possible for our users to send webhooks reliably at scale.
```

Comment:
This line talks about our docs, need to tie and it talk about why we want to make their docs easier too.

We care a lot of docs -> consider part of the product -> we created good docs -> we want to help you to create good docs.